README.md

```diff
@@ -52,7 +52,7 @@ Here are flows to try out, from simple to specific detailed variants.
 
 - **[Simple flow](READMEs/datatokens-flow.md)** - the essence of Ocean - creating a data NFT & datatoken.
```

Comment:
Given that this PR is about READMEs, another suggested tweak: rename file datatokens-flow.md to data-nfts-and-datatokens-flow.md. This is in line with similar changes in oceanprotocol.com and docs.